### PR TITLE
[IMP] l10n_ee: rework KMD and KMD INF reports

### PR DESCRIPTION
**Pull request by mairasalazar**
The tax report (KMD report) and annex to the tax report (KMD INF report) for Estonia needed to be reworked to correct errors and fix rounding issues.

In this commit, the 'balance_from_tags' label is added to the tax report, allowing a better audit of the report lines. Furthermore, the tax tags are edited to differentiate between tax and base tax lines.

These changes are necessary to improve auditing of the tax report lines.

task-3997203

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by diagnoza — APPROVED** [review]
Looks good from the upgrade pov.

**Review by hupo-odoo — COMMENTED** [review]
Good job, made one comment here and several in the enterprise PR

File: addons/l10n_ee/data/account_tax_report_data.xml [hunk 1]
```diff
@@ -56,7 +71,12 @@
                     <record id="tax_report_line_2_1_tag" model="account.report.expression">
                         <field name="label">balance</field>
                         <field name="engine">tax_tags</field>
-                        <field name="formula">2_1</field>
+                        <field name="formula">2_1_base</field>
```
[the remark this answers is not in the document]
Comment (mairasalazar) [reply addons/l10n_ee/data/account_tax_report_data.xml:75]: Indeed, looking at the [Estonian vat return form](https://www.emta.ee/sites/default/files/documents/2023-12/vorm_kmd_2024_eng.pdf), they should be at the same hierarchy, so changed it!

**Review by jbw-odoo — APPROVED** [review]
robodoo r+

**Pull request closed without merging**